### PR TITLE
Show apartment in tenant account.

### PR DESCRIPTION
**Pull request by Ayushi-0903**
I am not sure how to check if it is working or not.

### CODE REVIEW

**Comment by ab7289** [comment]
create an account and set yourself as the tenant for an apartment from the apartment detail view.  Then it should show up in in your account view.

File: mainapp/templates/account.html [hunk 1]
```diff
@@ -59,6 +59,57 @@ <h4 class=" mb-0">
         </div>
     </div>
 
+    <!-- Tenant Apartments -->
+    {% if user.tenant_apartment_set.count > 0 %}
+        <hr>
+        <h1>Tenant's Apartment</h1>
```
Comment (ab7289) [inline mainapp/templates/account.html]: Could we change this heading to an `<h2>` or `<h3>`? The `<h1>` is awfully large. Also could the text be chanced to "Places I live"?

Same thing for the "Landlord's Apartments" heading, could the h-tag be made smaller and the text changed to "Places I own"

**Comment by ab7289** [comment]
Please update both h tags.

Screen Shot 2019-11-25 at 6 13 29 PM

File: mainapp/templates/account.html (hunk 1 above)
Comment (Ayushi-0903) [reply mainapp/templates/account.html]: Screen Shot 2019-11-25 at 6 45 31 PM
I am getting this error while searching on local server. So I am unable to see the changes I am making.

**Comment by Ayushi-0903** [comment]
@ab7289 It is done.

File: location/templates/location.html [hunk 2]
```diff
@@ -40,6 +41,16 @@
 <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
 {% leaflet_map "leaflet_map_main" %}
 
+{% if request.user.is_authenticated %}
```
Comment (ab7289) [inline location/templates/location.html]: This looks like its from the favorite location story. Are you making sure to keep your changes between branches separate?

File: mainapp/templates/account.html [hunk 3]
```diff
@@ -59,10 +59,61 @@ <h4 class=" mb-0">
...
+        <!--Section: Cards-->
+    {% endif %}
+
+
+
     <!-- Landlord Apartments -->
     {% if user.landlord_apartment_set.count > 0 %}
         <hr>
-        <h1>Landlord Apartments</h1>
+        <h3>Places I own</h3>
```
Comment (ab7289) [inline mainapp/templates/account.html]: 👍

File: mainapp/templates/account.html [hunk 4]
```diff
@@ -59,10 +59,61 @@ <h4 class=" mb-0">
         </div>
     </div>
 
+    <!-- Tenant Apartments -->
+    {% if user.tenant_apartment_set.count > 0 %}
+        <hr>
+        <h3>Places I live</h3>
```
Comment (ab7289) [inline mainapp/templates/account.html]: 👍

File: location/templates/location.html (hunk 2 above)
Comment (Ayushi-0903) [reply location/templates/location.html]: Reverted changes.

**Comment by Ayushi-0903** [comment]
@ab7289 Check it now.

**Pull request merged**